client/packages/invoices/src/OutboundShipment/DetailView/OutboundLineEdit/OutboundLineEdit.tsx

```diff
@@ -244,6 +244,7 @@ export const OutboundLineEdit: React.FC<ItemDetailsModalProps> = ({
           packSizeController={packSizeController}
           onChangeItem={(item: ItemRowFragment | null) => {
             if (status === InvoiceNodeStatus.New) setIsDirty(true);
+            setIsAutoAllocated(false);
```

Comment:
This does most of the heavy lifting resetting the the issue quantities if you change the item... which honestly doesn't make that much sense to me? Is there a better name for what this state actually represents?